### PR TITLE
fix: continue Next() after a time.Time

### PR DESCRIPTION
**Pull request by lahickey**
A fix for https://github.com/tursodatabase/go-libsql/issues/19

### CODE REVIEW

File: libsql.go [hunk 1]
```diff
@@ -223,9 +224,6 @@ func openEmbeddedReplicaConnector(dbPath, primaryUrl, authToken string, readYour
 			}
 		}()
 	}
-	if err != nil {
```
Comment (haaawk) [inline libsql.go:226]: Why is this necessary?

**Review by haaawk — COMMENTED** [review]
LGTM in general. I'm not sure why the error handling was removed from openEmbeddedReplicaConnector

File: libsql.go (hunk 1 above)
Comment (lahickey) [reply libsql.go:226]: It isn't *necessary* but I took it out because it's redundant and the compiler will elide it anyway - as shown by the go linter. The error in that scope (at line 201 is already checked for nil (line 203) and the only other subsequent error is in the go routine (220) and never escapes the `if` or, indeed, the go routine.

I can add it back in and re-push if you like?

**Review by haaawk — APPROVED** [review]
